spec/dummy/db/seeds.rb

```diff
@@ -165,7 +165,15 @@
 25.times do
   post = FactoryBot.create(:post, user_id: users.sample.id)
 
-  post.cover_photo.attach(io: URI.open("https://source.unsplash.com/random/#{[1000, 1100, 1200, 1300].sample}x#{[1000, 1100, 1200, 1300].sample}/?all"), filename: "cover.jpg")
+  begin
+    width = [1000, 1100, 1200, 1300].sample
+    height = [1000, 1100, 1200, 1300].sample
+    url = "https://source.unsplash.com/random/#{width}x#{height}/?all"
+    io = URI.open(url) # standard:disable Security/Open
+  rescue
+    puts "Failed to fetch cover photo from Unsplash"
+  end
+  post.cover_photo.attach(io: io, filename: "cover.jpg") if io
```

Comment:
```suggestion
  begin
    width = [1000, 1100, 1200, 1300].sample
    height = [1000, 1100, 1200, 1300].sample
    url = "https://source.unsplash.com/random/#{width}x#{height}/?all"
    io = URI.open(url) # standard:disable Security/Open

    post.cover_photo.attach(io: io, filename: "cover.jpg")
  rescue
    puts "Failed to fetch cover photo from Unsplash"
  end
```

Reply:
Could this work as well?

Reply:
Yeah but in case just the attach fails the error message isn't true :)

Reply:
Got it! makes sense!
Ok. let's go with the original version.